### PR TITLE
support binary integers: 0b1101 literals and int("0b1101", 0)

### PR DESCRIPTION
**Pull request by MohamedElqdusy**

### CODE REVIEW

**Comment by MohamedElqdusy** [comment]
Checking the CLA

File: syntax/scan.go [hunk 1]
```diff
@@ -878,6 +891,7 @@ func isIdentStart(c rune) bool {
 func isdigit(c rune) bool  { return '0' <= c && c <= '9' }
 func isodigit(c rune) bool { return '0' <= c && c <= '7' }
 func isxdigit(c rune) bool { return isdigit(c) || 'A' <= c && c <= 'F' || 'a' <= c && c <= 'f' }
+func isbdigit(c rune) bool { return '0' <= c && c <= '1' }
```
Comment (alandonovan) [inline syntax/scan.go:894]: Simpler: c == '0' || c == '1'

File: syntax/scan.go [hunk 2]
```diff
@@ -770,6 +770,17 @@ func (sc *scanner) scanNumber(val *tokenValue, c rune) Token {
 				sc.readRune()
 				c = sc.peekRune()
 			}
+		} else if c == 'b' || c == 'B' {
```
Comment (alandonovan) [inline syntax/scan.go:773]: Could you add some tests to syntax/scan_test.go (for the scanner) and testdata/int.sky for the int(x) function?

**Review by alandonovan — COMMENTED** [review]
Many thanks for contributing!  The code change looks good, but we should add tests and a line in the spec.md documentation.

**Comment by MohamedElqdusy** [comment]
Thank you for your fast responding to review, I have added the new changes to #52

**Pull request closed without merging**